script/tool/lib/src/create_all_plugins_app_command.dart

```diff
@@ -179,7 +205,7 @@ class CreateAllPluginsAppCommand extends PackageCommand {
       },
       dependencyOverrides: pluginDeps,
     );
-    app.pubspecFile.writeAsStringSync(_pubspecToString(pubspec));
+    app.pubspecFile.writeAsStringSync(_pubspecToString(pubspec), flush: true);
```

Comment:
Oops, I didn't mean to push this; I was trying it locally when I was grasping at straws on the `pub get` hang issue. You can revert this line.